testpilot/frontend/static-src/app/views/experiment-page.js

```diff
@@ -273,5 +275,6 @@ export default PageView.extend({
         { value: 'other', title: 'feedbackAnswerOther' }
       ]
     }), 'body');
+    app.sendToGA('event', {eventCategory: 'ExperimentDetailsPage Interactions', eventAction: 'button click', eventLabel: 'Give Feedback'});
```

Comment:
We usually like to keep our lines around 80 chars (like pep8 in python) it's not a hard rule, just makes files easier to parse. In this case I would break this into a couple different lines. For example: 
``` js
app.sendToGA('event', {
  eventCategory: 'ExperimentDetailsPage Interactions',
  eventAction: 'button click',
  eventLabel: 'Give Feedback'
});
```